### ISSUE TITLE
Which microphone do you use?

### ISSUE THREAD

**Issue by jeremyfritzen** [ivadim/fruitnanny#25]
After having tested my installation (proof of concept), I noticed that the cheap microphone I got has a very low quality. It's actually not usable at all (almost can't hear anything but noise). It was [this one](https://www.amazon.fr/gp/product/B071YMZQP1/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1).

Which microphone do you recommend to use?
Of course, I am looking for something as cheap and little as possible.

Thank you!

**Comment by bryanberger** [comment]
@jeremyfritzen Some people had luck returning that microphone over-and-over to Amazon until they get a replacement that works better.

I've seen people recommend this condenser mic on other threads: [Samson-Mic-Portable-Condenser-Microphone]( https://www.amazon.com/Samson-Mic-Portable-Condenser-Microphone/dp/B001R76D42/ref=sr_1_43?keywords=usb+microphone&qid=1578069569&sr=8-43) but it is much larger and more expensive.

You might also have better luck with a gooseneck, omnidirectional, or lavalier mic.

You can also adjust the input/output gain of your microphone from the raspberrypi by typing in `alsamixer` into the console and adjusting your mic.

It will look something like this:
![Screen Shot 2020-01-03 at 11 46 37 AM](https://user-images.githubusercontent.com/940022/71736308-bd6aa180-2e1e-11ea-9e6e-13dd2ea21122.png)

I personally use an old [Blue Yeti ](https://www.amazon.com/Blue-Yeti-USB-Microphone-Blackout/dp/B00N1YPXW2/ref=sr_1_6?keywords=blue+yeti+nano&qid=1578070057&sr=8-6)that I had laying around (totally overkill for this, but delivers excellent audio. I just run a usb extension and mount it under the crib)

**Comment by jeremyfritzen** [comment]
Thanks, it helps pretty well!
I already tried to adjust the input/output gain but it doesn't change anything...

By the way, did you change the project a little bit in order to be notified where there is some noise?

**Comment by bryanberger** [comment]
@jeremyfritzen no, I just leave it on all the time on an iPad, since I use a good quality mic, I can hear a pin drop -- no need for notifications.

**Comment by jeremyfritzen** [comment]
Indeed, pretty convenient ^^

**Comment by ivadim** [comment]
Hi @jeremyfritzen 
I've tried several cheap mics from amazon and made return some of them due to high level of noise and some because they didn't work at all. Right now I use this one https://www.amazon.com/gp/product/B00XU1GHO4/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1

What I actually noticed, noise comes not from the mic itself but from PSU (power supply unit) and better the power device you have less noise you hear. So I had problems like you when I used cheap charger from aliexpress. I switched later to one from my router and noise level went significantly down (but still exists). 

Additionally, make sure your PSU has Ferrite filter: https://en.wikipedia.org/wiki/Ferrite_bead . This should reduce static noise level as well. And I'm going to merge #22 , hope it will help

**Comment by ivadim** [comment]
I merged #22 and it looks like helped a lot. I would appreciate if you can test it 🙇
